### ISSUE TITLE
Fail to build rustix with nightly

### ISSUE THREAD

**Issue by rllola** [bytecodealliance/rustix#354]
I was trying to install wizer and failed to install it.

I used the command :
```
cargo install wizer --all-features
```
with `cargo 1.62.0-nightly (1ef1e0a 2022-03-31)`

and got the error
```
error: cannot find macro `asm` in this scope
   --> /home/lola/.cargo/registry/src/github.com-1ecc6299db9ec823/rustix-0.26.2/src/imp/linux_raw/arch/inline/x86_64.rs:295:5
    |
295 |     asm!(
    |     ^^^
    |
    = note: consider importing one of these items:
            std::arch::asm
            core::arch::asm

error: could not compile `rustix` due to 14 previous errors
warning: build failed, waiting for other jobs to finish...
error: failed to compile `wizer v1.3.5`, intermediate artifacts can be found at `/tmp/cargo-installedMD7S`
```

Using rust 1.61 fixed my problem.

**Comment by sunfishcode** [comment]
It looks like this is fixed in wizer; I've filed https://github.com/bytecodealliance/wizer/issues/51 to ask about publishing a new version to crates.io.

**Comment by sunfishcode** [comment]
This is now fixed in wizer 1.4.
